src/util/Constants.js

```diff
@@ -431,6 +434,7 @@ exports.InviteScopes = [
  * * GUILD_DISCOVERY_DISQUALIFIED
  * * GUILD_DISCOVERY_REQUALIFIED
  * * REPLY
+ * * APPLICATION_COMMAND
  * @typedef {string} MessageType
  */
 exports.MessageTypes = [
```

Comment:
below this is the `SystemMessageTypes` export, that needs to be updated to exclude the new type